### ISSUE TITLE
Unauthorized Error (Trying to set up a demo system)

### ISSUE THREAD

**Issue by jeromerg** [ingef/conquery#2618]
Hi together,

I am trying to set up a demo environment. I compiled and started both backend and frontend locally, then executed the demo python notebook to fetch, transform and upload the data to the backend. Everything works fine. In the Admin UI, I can see the dataset. But when I open the UI, it intends to fetch some information from the backend, including the list of datasets, but the server replies with a 401 (Unauthorized) Status Code. 

Can you explain shortly how to set up a user account with valid credentials, so that I can further test the software?

Thank you in advanced!
Kind Regards

**Comment by Flou21** [comment]
I have the same issue

**Comment by thoniTUB** [comment]
Hey,

> tldr; open the frontend with `http://localhost:8000?access_token=user.SUPERUSER`

the default configuration uses the [DevelopmentAuthorizationConfig](https://github.com/ingef/conquery/blob/70976c7e394b1cd98bb6b9a9f2748bdde27bd7f7/backend/src/main/java/com/bakdata/conquery/models/config/auth/DevelopmentAuthorizationConfig.java#L28) with a superuser configured and the [FirstInitialUserRealm](https://github.com/ingef/conquery/blob/develop/backend/src/main/java/com/bakdata/conquery/models/auth/develop/FirstInitialUserRealm.java).  
The latter simply tries to extract a user id from the authorization header of a request and looks up the user.  
If no user was found, the acces is denied.  When no header was given, simply the first configured user is assumed for the request.

The frontend always sends an authorization header, even if it is empty (like `Authorization: Bearer `). You can fill the missing part by opening the frontend with the `access_token` query parameter set: `http://localhost:8000?access_token=user.SUPERUSER`.
Then all frontend request will have the header set like this: `Authorization: Bearer user.SUPERUSER`.

The DevelopmentAuthorizationConfig recently changed the superuser id. It was `user.SUPERUSER@SUPERUSER` before. I'll look into the notebook if the change was propaged there.

**Comment by thoniTUB** [comment]
I checked the [notebook](https://github.com/ingef/conquery/blob/70976c7e394b1cd98bb6b9a9f2748bdde27bd7f7/tutorial/mimic_iii_demo/preprocess_and_upload.ipynb).

The last cell contains the correct user id and intructions for the frontend.
